charmcraft/utils.py

```diff
@@ -20,8 +20,9 @@
 import os
 import pathlib
 import platform
+import sys
 from collections import namedtuple
-from stat import S_IXUSR, S_IXGRP, S_IXOTH, S_IRUSR, S_IRGRP, S_IROTH
+from stat import S_IRGRP, S_IROTH, S_IRUSR, S_IXGRP, S_IXOTH, S_IXUSR
```

Comment:
I would appreciate these more close to release delicate PRs not having these type of changes.

Reply:
This is probably isort in action.

Reply:
Yes it is.